README.md

```diff
@@ -18,6 +18,16 @@
 6. Go to http://localhost:3000/explorer-next to get an overview of available endpoints and database schemas.
 7. To be able to run the e2e tests with the same setup as in the Github actions you will need to run `npm run  prepare:local` and after that run `npm run start:dev`. This will start all needed containers and copy some configuration to the right place.
 
+## Develop in a container using the docker-compose.dev file
+
+1. `git clone https://github.com/SciCatProject/scicat-backend-next.git`
+2. docker-compose -f docker-compose.dev.yaml up -d
+3. *Optional* Mount *functionalAccounts.json* file to a volume in the container. If not set up, the functional accounts in [functionalAccounts.json.example](/functionalAccounts.json.example) will be created automatically.
+4. *Optional* change the container env variables
+5. Attach to the container
+6. `npm run start:dev`
```

Comment:
One thing that I don't like about the process is having to run the "npm run start:dev" after the docker containers have been set up. This manual step won't deploy nicely to production. I have tried putting this command in the docker file and in the docker compose file under the command part. However neither work. It would be nice to have a set up that we can spin up in docker-compose without any other intervention.

Reply:
So in my above comment I have got it running with the backend from `main` which is a different dockerfile. I can't get it to work with the docker file in this PR.

Reply:
ok, that could be. I have the feeling the breaking bit is [this](https://github.com/SciCatProject/scicat-backend-next/pull/153/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R1). I had to add it because on MAC that is needed, but it could be that it complains on other architectures (I couldn't try it), and I forgot to add such a comment. Could you please try without it and let me know if it works? or maybe you could post here the error logs. Thanks!